src/main/java/teammates/common/datatransfer/UserInfoCookie.java

```diff
@@ -14,9 +17,12 @@
     private String userId;
     private String verificationCode;
 
+    private Instant expiryTime;
+
     public UserInfoCookie(String userId) {
         this.userId = userId;
         this.verificationCode = StringHelper.generateSignature(userId);
+        this.expiryTime = Instant.now().plus(7, ChronoUnit.DAYS);
```

Comment:
`7` is a magic number repeated here and in one of the servlet classes (where the actual cookie age is set)